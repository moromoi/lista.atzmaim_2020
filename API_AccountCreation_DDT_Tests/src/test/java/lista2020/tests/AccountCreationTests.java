package lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.annotations.Test;

public class AccountCreationTests extends TestBase {

    @Test(priority = 1)
    public void getBusinessTypeJSON() {

        Approvals.verify(app.accountCreationHelper().businessTypeResponse());

    }

    @Test(priority = 2)
    public void createAccount() {
        app.accountCreationHelper().createAccount();
    }

    @Test(priority = 3)
    public void deleteAccount() {
        app.accountCreationHelper().deleteAccount();
    }
}
