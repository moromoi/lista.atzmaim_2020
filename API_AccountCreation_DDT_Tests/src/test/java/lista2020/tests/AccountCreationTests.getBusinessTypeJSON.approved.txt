[{"id":1,"name":"Holistic Therapists","icon":"alternative_therapy_specialists.svg","ordering":1},{"id":2,"name":"Consultants and life \/ professional coaches","icon":"consultants_and_coachers.svg","ordering":1},{"id":3,"name":"Cosmetics & Beauty Specialists","icon":"cosmetics_and_beauty_therapists.svg","ordering":1},{"id":4,"name":"Hair Salon and Barbers","icon":"hair_stylists_and_salons.svg","ordering":1},{"id":5,"name":"Interior designers and architects","icon":"interior_designers_and_architects.svg","ordering":1},{"id":6,"name":"Installers & technicians","icon":"installers_and_technicians.svg","ordering":1},{"id":7,"name":"Make up artists","icon":"makeup_artists.svg","ordering":1},{"id":8,"name":"Nail Artists","icon":"manicurists_and_nail_artists.svg","ordering":1},{"id":9,"name":"Massage and Reflexology Therapists","icon":"massage_and_reflexology_therapists.svg","ordering":1},{"id":10,"name":"Movers and Cleaners","icon":"movers_and_cleaners.svg","ordering":1},{"id":11,"name":"Nutritionists & Naturopaths","icon":"nutritionists_and_naturopaths.svg","ordering":1},{"id":12,"name":"Pest controllers","icon":"pest_controllers.svg","ordering":1},{"id":13,"name":"Teachers and tutors","icon":"private_teachers_and_tutors.svg","ordering":1},{"id":14,"name":"Sales\/ Real Estate \/ Insurance Agents","icon":"real_estate_agents_marketers_insurers.svg","ordering":1},{"id":15,"name":"Service providers for home and business","icon":"service_providers_for_home_and_business.svg","ordering":1},{"id":16,"name":"Personal Trainers","icon":"sports_and_fitness_instructors.svg","ordering":1},{"id":17,"name":"Tattoo and piercing shops","icon":"tattoo_artists.svg","ordering":1},{"id":18,"name":"Therapists","icon":"therapy_specialists.svg","ordering":1},{"id":19,"name":"Holistic Therapists","icon":"alternative_therapy_specialists.svg","ordering":1},{"id":20,"name":"Consultants and life \/ professional coaches","icon":"consultants_and_coachers.svg","ordering":1},{"id":21,"name":"Cosmetics & Beauty Specialists","icon":"cosmetics_and_beauty_therapists.svg","ordering":1},{"id":22,"name":"Hair Salon and Barbers","icon":"hair_stylists_and_salons.svg","ordering":1},{"id":23,"name":"Interior designers and architects","icon":"interior_designers_and_architects.svg","ordering":1},{"id":24,"name":"Installers & technicians","icon":"installers_and_technicians.svg","ordering":1},{"id":25,"name":"Make up artists","icon":"makeup_artists.svg","ordering":1},{"id":26,"name":"Nail Artists","icon":"manicurists_and_nail_artists.svg","ordering":1},{"id":27,"name":"Massage and Reflexology Therapists","icon":"massage_and_reflexology_therapists.svg","ordering":1},{"id":28,"name":"Movers and Cleaners","icon":"movers_and_cleaners.svg","ordering":1},{"id":29,"name":"Nutritionists & Naturopaths","icon":"nutritionists_and_naturopaths.svg","ordering":1},{"id":30,"name":"Pest controllers","icon":"pest_controllers.svg","ordering":1},{"id":31,"name":"Teachers and tutors","icon":"private_teachers_and_tutors.svg","ordering":1},{"id":32,"name":"Sales\/ Real Estate \/ Insurance Agents","icon":"real_estate_agents_marketers_insurers.svg","ordering":1},{"id":33,"name":"Service providers for home and business","icon":"service_providers_for_home_and_business.svg","ordering":1},{"id":34,"name":"Personal Trainers","icon":"sports_and_fitness_instructors.svg","ordering":1},{"id":35,"name":"Tattoo and piercing shops","icon":"tattoo_artists.svg","ordering":1},{"id":36,"name":"Therapists","icon":"therapy_specialists.svg","ordering":1}]